### PR TITLE
feat(docker): add local fs docker compose

### PR DESCRIPTION
**Pull request by wcy-fdu**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
as title, verified.

rated to https://github.com/risingwavelabs/risingwave-docs/issues/1635

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: docker/docker-compose-with-local-fs.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                    --config-path /risingwave.toml\" \
+                 --compute-opts=\" \
+                    --config-path /risingwave.toml \
+                    --listen-addr 0.0.0.0:5688 \
+                    --prometheus-listener-addr 0.0.0.0:1222 \
+                    --advertise-addr 0.0.0.0:5688 \
+                    --async-stack-trace verbose \
+                    --connector-rpc-endpoint 0.0.0.0:50051 \
+                    #--parallelism 4 \
+                    #--total-memory-bytes 8589934592 \
```
Comment (MrCroxx) [inline docker/docker-compose-with-local-fs.yml:27]: Remove or leave some comments?

File: docker/docker-compose-with-local-fs.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    <<: *image
+    command: "standalone --meta-opts=\" \
+                    --listen-addr 0.0.0.0:5690 \
+                    --advertise-addr 0.0.0.0:5690 \
+                    --dashboard-host 0.0.0.0:5691 \
+                    --prometheus-host 0.0.0.0:1250 \
+                    --connector-rpc-endpoint 0.0.0.0:50051 \
+                    --backend etcd \
+                    --etcd-endpoints etcd-0:2388 \
+                    --state-store hummock+fs://<loacl-path> \
```
Comment (MrCroxx) [inline docker/docker-compose-with-local-fs.yml:16]: typo

```suggestion
                    --state-store hummock+fs://<local-path> \
```

File: docker/docker-compose-with-local-fs.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        - bash -c 'printf \"GET / HTTP/1.1\n\n\" > /dev/tcp/127.0.0.1/5690; exit $$?;'
+      interval: 1s
+      timeout: 5s
+    restart: always
+    deploy:
+      resources:
+        limits:
+          memory: 28G
+        reservations:
+          memory: 28G
```
Comment (MrCroxx) [inline docker/docker-compose-with-local-fs.yml:85]: Why `28G`?

**Review by MrCroxx — APPROVED** [review]
LGTM

**Review by hzxa21 — APPROVED** [review]

**Pull request merged**